resources/apigateway-domainnames.go

```diff
@@ -43,10 +43,25 @@ func (l *APIGatewayDomainNameLister) List(ctx context.Context, o interface{}) ([
 
 		for i := range output.Items {
 			item := &output.Items[i]
+			
```

Comment:
Please create a map[string]string and initialize up here, then if `tagsOutput` is not nil, you can set to this variable and then reference it in the `resources = append` section. 

Reason: we do not want to error on getting tags, so we need to handle empty vs not empty.